bson/primitive/decimal.go

```diff
@@ -114,6 +121,80 @@ Loop:
 	return string(repr[last+pos:])
 }
 
+// BigInt returns significand as big.Int and exponent, bi * 10 ^ exp.
+func (d Decimal128) BigInt() (bi *big.Int, exp int, err error) {
+	h, l := d.GetBytes()
+	var pos int // positive sign
```

Comment:
Hmmm, it seems like `pos` is used exactly like a `bool`, so I'm thinking it should be a `bool`. :slightly_smiling_face: